dubbo-config/dubbo-config-spring/src/main/resources/META-INF/dubbo.xsd

```diff
@@ -387,6 +387,11 @@
                 <xsd:documentation><![CDATA[ Is default. ]]></xsd:documentation>
             </xsd:annotation>
         </xsd:attribute>
+        <xsd:attribute name="enable-telnet" type="xsd:string">

```

Comment:
There are two dubbo.xsd file in the project.
Pls copy to another one.

Reply:
already do it.